app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java

```diff
@@ -216,51 +216,74 @@
 
   private BookmarksDao bookmarksDao;
 
-  @BindView(R.id.toolbar) Toolbar toolbar;
+  @BindView(R.id.toolbar)
```

Comment:
You've submitted way too much - please use git patch mode to only submit actual changes, and no changes caused by you using a different auto format config.